tflite/python/tflitePythonVivante/.conf/deps.json

```diff
@@ -5,8 +5,6 @@
         "python3",
         "python3-pip",
         "python3-pylint-common",
-        "python3-venv",
-        "python3-numpy",
```

Comment:
I think we need to keep `python3-numpy` here, since the application needs it.

Reply:
@andreriesco, I think numpy is installed with Tensoflow Lite here: https://github.com/toradex/torizon-samples/pull/18/files/814c551748f5b5ce5760fdd97b3a6063ef3da238#diff-7ce7be0f090719e6ecb5560365d3c08aea233046719f83a4e074b9267e2c2d1fR96

It is one of the dependencies of the TensorFlow lite wheel: https://github.com/nxp-imx/tensorflow-imx/blob/lf-5.15.71_2.2.0/tensorflow/lite/python/BUILD#L63

Reply:
Actually this packages present in the `.vscode/deps.json` are the ones that are installed on the host machine. So, as the application needs numpy to be executed on Local Debug, it would need numpy to be installed on the host

Reply:
@andreriesco, I believe that what is installed on the host when you try to run the project locally is the contents of the requirements files.
It even does that to the local python system packages with `/bin/python3 -m pip install --break-system-packages -r requirements-debug.txt`, which is quite dangerous.

Given that it is this way, there are two paths forward:
- Add the required dependencies to the `requirements*.txt` files and test if this does not break the container on the module.
- Only support the containers to run on the module

My preference would be to only support the containers to run on the module, as such installation of packages to the local python system packages is not acceptable in my opinion.

Reply:
The numpy in specific is actually being installed through an `apt` package [on the container](https://github.com/toradex/torizon-samples/pull/18/commits/814c551748f5b5ce5760fdd97b3a6063ef3da238#diff-be950562ae9442005cab237cc42705ee3bd9c2c999116268c9b2c17aeea14351R41), so that's why it is on `deps.json`. 
About the installation of the pip packages with this command `/bin/python3 -m pip install --break-system-packages -r requirements-debug.txt`, probably we should review it and improve this. But not just for this sample, for the whole python3 IDE template. Can you open a thread about this on the Slack channel of the IDE so we can discuss better this problem and possible solutions, please?

Reply:
Sounds ok for now to keep the usage just inside the containers. Then, after we change the default behavior of the Python template I believe we can update the sample with this change and then allow the local debugging also

Reply:
Hello @andreriesco,

Sorry for bringing back this topic, I was working on enabling the local debugging and did not see your comment.

I added support for it with the following commit: https://github.com/toradex/torizon-samples/pull/18/commits/e05f7a325bb183dbcc3ce48abd02a734352a7a63

It still is not ideal if the user tries to run it locally because of the issues we discussed, but it should work without problems within a venv.

Please let me know if you would like me to revert the commit or if we can keep it as is.

Reply:
Note: the addition of [requirements-debug-local.txt](https://github.com/toradex/torizon-samples/commit/e05f7a325bb183dbcc3ce48abd02a734352a7a63#diff-094433cd1023e5e21c0e5c930da22e613a18f6e7e8d2fd1d243924033f617be9) is intended to avoid any unnecessary install of packages in the container.

Reply:
Nice, I think we can keep it as it is. Probably I will something similar to this on the python3 IDE template, so I think it makes sense to keep it.